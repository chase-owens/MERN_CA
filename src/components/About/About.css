.scrollContainer::-webkit-scrollbar {
  display: none;
}
