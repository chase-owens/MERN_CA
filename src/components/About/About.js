import React from 'react';

import Grid from '@material-ui/core/Grid';
import Typography from '@material-ui/core/Typography';
import Card from '@material-ui/core/Card';
import withWidth from '@material-ui/core/withWidth';
import { withStyles } from '@material-ui/core/styles';
import { unstable_useMediaQuery as useMediaQuery } from '@material-ui/core/useMediaQuery';

import { theme } from '../../styles/theme';
import css from './About.css';

import aboutPoints from './about.config';
import targetOutcomes from './targetOutcomes.config';
import AboutCardSmall from './AboutCardSmall';
import TargetOutcomesSmall from './TargetOutcomesSmall';

const getBackground = i => {
  switch (i) {
    case 0:
      return theme.palette.primary.main;
    case 1:
      return theme.palette.ternary.dark;
    case 2:
      return theme.palette.secondary.main;
    default:
      return null;
  }
};

const About = ({ theme }) => {
  console.log(aboutPoints);
  return (
    <div>
      {useMediaQuery(theme.breakpoints.down(740)) && (
        <div>
          <div
            style={{
              margin: '0 30px 60px 40px'
            }}
          >
            <Typography paragraph variant='headline'>
              The plan: Create calm connections with affection, love, and praise
            </Typography>
            <Typography paragraph variant='body1'>
              If your dog is always jumping on you, nipping your hands, or
              biting your clothes to get your attention, don’t worry - That’s a
              good thing! That means your dog loves you, that your affection has
              already been established as a reinforcer, and that you’re ready to
              begin Constructional Affection.{' '}
            </Typography>
          </div>
          <br />
          <Grid
            container
            className='scrollContainer'
            wrap='nowrap'
            style={{
              overflowX: 'auto',
              WebkitOverflowScrolling: 'touch'
            }}
            spacing={40}
          >
            {aboutPoints !== undefined &&
              aboutPoints.map((point, i) => {
                let background = getBackground(i);
                let color = i % 2 == 0 ? 'textPrimary' : 'textSecondary';
                let margin = i % 2 === 1 ? 36 : 0;
                return (
                  <Grid
                    item
                    key={point.title}
                    style={{
                      flexShrink: 0,
                      width: '65%',
                      height: '65vw',
                      minHeight: 450,
                      minWidth: 400
                      // background: background,
                    }}
                  >
                    <Card
                      style={{
                        width: '100%',
                        height: '100%',
                        // minHeight: 400,
                        // minWidth: 400,
                        background: background
                      }}
                    >
                      <AboutCardSmall point={point} color={color} />
                    </Card>
                  </Grid>
                );
              })}
          </Grid>
          <br />
          <Grid
            container
            className='scrollContainer'
            wrap='nowrap'
            style={{
              overflowX: 'auto',
              WebkitOverflowScrolling: 'touch'
            }}
            spacing={40}
          >
            {targetOutcomes !== undefined &&
              targetOutcomes.map(target => (
                <Grid
                  item
                  key={target.title}
                  style={{
                    flexShrink: 0,
                    width: '65%',
                    height: '65vw',
                    minHeight: 400,
                    minWidth: 400
                    // background: background,
                    // marginLeft: margin,
                    // marginRight: margin
                  }}
                >
                  <Card
                    style={{
                      width: '100%',
                      height: '100%'
                      // minHeight: 400,
                      // minWidth: 400,
                      // background: background
                    }}
                  >
                    <TargetOutcomesSmall
                      title={target.title}
                      steps={[target.a, target.b, target.c]}
                    />
                  </Card>
                </Grid>
              ))}
          </Grid>
          <br />
        </div>
      )}
      {useMediaQuery(theme.breakpoints.up(740)) && (
        <Grid container direction='row'>
          <Grid item sm='9' lg='6' style={{ padding: '0 10px' }}>
            <Typography
              variant='h2'
              style={{
                marginLeft: 30,
                fontSize: '2.5em'
              }}
            >
              The plan: Create calm connections with affection, love, and praise
            </Typography>
            <Typography variant='body1'>
              If your dog is always jumping on you, nipping your hands, or
              biting your clothes to get your attention, don’t worry - That’s a
              good thing! That means your dog loves you, that your affection has
              already been established as a reinforcer, and that you’re ready to
              begin Constructional Affection.{' '}
            </Typography>
          </Grid>
          <Grid
            item
            sm='9'
            lg='6'
            style={{ background: theme.palette.ternary.main }}
          >
            <Typography
              variant='h2'
              style={{
                fontSize: '2.5em'
              }}
            >
              Traditional training approaches often fail to solve the problem.
            </Typography>
          </Grid>
        </Grid>
      )}
    </div>
  );
};

export default withWidth({ withTheme: true })(About);
