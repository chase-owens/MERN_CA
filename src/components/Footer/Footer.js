import React, { useState } from 'react';
import { theme } from '../../styles/theme';
import { withStyles } from '@material-ui/core/styles';
import API from 'utils/api';

import Typography from '@material-ui/core/Typography';
import TextField from '@material-ui/core/TextField';
import Button from '@material-ui/core/Button';

const styles = theme => ({
  footer: {
    // width: '100%',
    minHeight: 350,
    background: theme.palette.primary.main,
    padding: 25
  },
  formContainer: {
    width: '80%',
    maxWidth: 400,
    margin: 'auto'
  },
  formHeader: {
    paddingTop: 30,
    fontSize: '2em'
  },
  formInputs: {
    margin: 'auto',
    width: 300
  },
  formInput: {
    marginBottom: 13
  },
  buttonContainer: {
    width: 87,
    margin: 'auto'
  },
  button: {
    margin: 'auto',
    background: theme.palette.ternary.main,
    color: '#fff',
    marginTop: 15
  }
});

const Footer = ({ classes }) => {
  const [name, setName] = useState('');
  const [email, setEmail] = useState('');
  const [message, setMessage] = useState('');

  const sendData = () => {
    const data = { name, email, message };
    console.log(data);
    API.post('/contact', data)
      .then(res => {
        // resetForm();
        console.log(res);
      })
      .catch(err => console.log('Message not sent', err));
    resetForm();
  };

  const resetForm = () => {
    setName('');
    setEmail('');
    setMessage('');
  };

  return (
    <footer className={classes.footer}>
      <div className={classes.formContainer}>
        <Typography
          className={classes.formHeader}
          align='center'
          paragraph
          variant='display4'
        >
          Contact Us
        </Typography>
        <div className={classes.formInputs}>
          <div className={classes.formInput}>
            <TextField
              style={{ width: '100%' }}
              label='Name'
              placeholder='Name'
              onChange={e => setName(e.target.value)}
              value={name}
            />
          </div>
          <div className={classes.formInput}>
            <TextField
              style={{ width: '100%' }}
              label='Email'
              placeholder='Email'
              onChange={e => setEmail(e.target.value)}
              value={email}
            />
          </div>
          <div className={classes.formInput}>
            <TextField
              style={{ width: '100%' }}
              label='Comment'
              multiline
              rowsMax='4'
              placeholder='Comment'
              onChange={e => setMessage(e.target.value)}
              value={message}
            />
          </div>
        </div>
        <div className={classes.buttonContainer}>
          <Button
            className={classes.button}
            onClick={sendData}
            variant='contained'
          >
            Submit
          </Button>
        </div>
      </div>
    </footer>
  );
};

export default withStyles(styles)(Footer);
