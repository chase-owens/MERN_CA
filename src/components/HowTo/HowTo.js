import React, { useRef } from 'react';
import { theme } from '../../styles/theme';
import caSteps from './caSteps.config';
import { isMobile } from 'react-device-detect';
import withWidth from '@material-ui/core/withWidth';
import { unstable_useMediaQuery as useMediaQuery } from '@material-ui/core/useMediaQuery';

import Grid from '@material-ui/core/Grid';
import Typography from '@material-ui/core/Typography';
import InteractionGuidelines from '../InteractionGuidelines/InteractionGuidelines';
import AffectionLoop from '../AffectionLoop/AffectionLoop';
import InfoGraphic from '../InfoGraphic/InfoGraphic';

const HowTo = () => {
  const howTo = useRef(null);

  return isMobile ? (
    <div id='howTo' ref={howTo} style={{ paddingTop: 20 }}>
      <Typography
        id='howTo'
        paragraph
        variant='caption'
        style={{ marginLeft: 100, fontSize: '2em', marginBottom: 45 }}
      >
        <span style={{ fontWeight: 'bold' }}>CA</span>/ How To
      </Typography>

      <Grid
        container
        className='scrollContainer'
        wrap='nowrap'
        style={{
          overflowX: 'auto',
          WebkitOverflowScrolling: 'touch',
          flex: '0 1 auto',
          maxWidth: 1000,
          margin: 'auto',
          marginBottom: 70,
          marginTop: 30
        }}
        spacing={16}
        justify='space-around'
      >
        {caSteps.map(step => (
          <Grid item key={step.title}>
            <InfoGraphic info={step} />
          </Grid>
        ))}
      </Grid>
      <Grid
        container
        spacing={0}
        justify='space-between'
        style={{
          width: '100vw'
        }}
      >
        <Grid
          item
          style={{
            width: '100vw'
          }}
        >
          <InteractionGuidelines />
        </Grid>

        <Grid
          item
          style={{
            width: '100vw'
          }}
        >
          <AffectionLoop />
        </Grid>
      </Grid>
    </div>
  ) : (
    <div id='howTo' ref={howTo} style={{ paddingTop: 20 }}>
      <Typography paragraph variant='caption' style={{ paddingLeft: 40 }}>
        <span style={{ fontWeight: 'bold' }}>CA</span> / How To
      </Typography>
      <Grid
        container
        className='scrollContainer'
        wrap='nowrap'
        style={{
          overflowX: 'auto',
          WebkitOverflowScrolling: 'touch',
          flex: '0 1 auto',
          paddingLeft: useMediaQuery(theme.breakpoints.down('xs')) ? 100 : null,
          maxWidth: 800,
          margin: 'auto',
          marginBottom: 70,
          marginTop: 30,
          width: '100%'
        }}
        spacing={16}
        justify='space-around'
      >
        {caSteps.map(step => (
          <Grid item key={step.title}>
            <InfoGraphic info={step} />
          </Grid>
        ))}
      </Grid>
      <Grid
        container
        spacing={0}
        justify='space-between'
        style={{
          background: useMediaQuery(theme.breakpoints.up('740'))
            ? `linear-gradient(90deg, ${theme.palette.ternary.main} 50%, ${
                theme.palette.secondary.main
              } 50%)`
            : null,
          width: '100%'
        }}
      >
        <Grid
          item
          style={{
            width: useMediaQuery(theme.breakpoints.down('740')) ? '100%' : '50%'
          }}
        >
          <InteractionGuidelines />
        </Grid>

        <Grid
          item
          style={{
            width: useMediaQuery(theme.breakpoints.down('740')) ? '100%' : '50%'
          }}
        >
          <AffectionLoop />
        </Grid>
      </Grid>
    </div>
  );
};

export default withWidth({ withTheme: true })(HowTo);
