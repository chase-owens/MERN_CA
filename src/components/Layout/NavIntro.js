import React from "react";
import Nav from "../Nav/Nav";
import IntroDiv from "../IntroDiv/IntroDiv";
import { connect } from "react-redux";

import withWidth from "@material-ui/core/withWidth";
import useMediaQuery from "@material-ui/core/useMediaQuery";

import { theme } from "../../styles/theme";

import { isMobile } from "react-device-detect";

const mapStateToProps = (state) => {
  return {
    language: state.languageState.language,
    open: state.sideBarState.open,
  };
};

const NavIntro = ({ theme }) => {
  // return isMobile ? (
  //   <div>
  //     <nav>
  //       <Nav />
  //     </nav>
  //     <IntroDiv />
  //   </div>
  // ) : (
  return (
    <div
      style={{
        marginLeft: 0,
        paddingLeft: 0,
        background: !useMediaQuery(theme.breakpoints.up(740))
          ? "#fff"
          : `linear-gradient(90deg, #fff 40%, ${theme.palette.primary.main} 40%)`,
      }}
    >
      <nav>
        <Nav />
      </nav>
      <IntroDiv />
    </div>
  );
};

export default withWidth({ withTheme: true })(
  connect(mapStateToProps)(NavIntro)
);
