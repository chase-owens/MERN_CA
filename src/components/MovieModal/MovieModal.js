import React from 'react';
import ReactPlayer from 'react-player';

import Icon from '@material-ui/core/Icon';
import Button from '@material-ui/core/Button';
import Grid from '@material-ui/core/Grid';

import withWidth from '@material-ui/core/withWidth';
import { unstable_useMediaQuery as useMediaQuery } from '@material-ui/core/useMediaQuery';

import { connect } from 'react-redux';
import { bindActionCreators } from 'redux';
import { toggleMovie } from '../Videos/videos.action';

import { theme } from 'styles/theme';

const mapStateToProps = state => ({
  focusedVideo: state.videoState.video
});

const mapDispatchToProps = dispatch =>
  bindActionCreators({ toggleMovie }, dispatch);

const MovieModal = ({ focusedVideo, toggleMovie }) => {
  return (
    focusedVideo !== null && (
      <div
        style={{
          width: '100vw',
          height: '100vh',
          // opacity: 0.7,
          // background: theme.palette.ternary.main,
          position: 'fixed',
          top: 0
        }}
      >
        <div
          style={{
            width: '100vw',
            height: '100vh',
            opacity: 0.7,
            background: theme.palette.ternary.main,
            position: 'relative',
            top: 0
          }}
        >
          {useMediaQuery(theme.breakpoints.up(740)) && (
            <Grid container justify='flex-end'>
              <Grid item>
                <Button
                  style={{
                    position: 'fixed',
                    color: '#fff',
                    zIndex: 999,
                    top: 3,
                    right: 3
                  }}
                  onClick={() => toggleMovie(null)}
                >
                  <Icon
                    style={{
                      paddingBottom: 15,
                      paddingTop: -15,
                      paddingRight: 20,
                      marginRight: -12,
                      marginLeft: -5
                    }}
                  >
                    <i style={{ fontSize: 40 }} class='material-icons'>
                      close
                    </i>
                  </Icon>
                </Button>
              </Grid>
            </Grid>
          )}
        </div>
        <div
          style={{
            width: '90vw',
            height: '90vh',
            // opacity: 0.7,
            // background: theme.palette.ternary.main,
            position: 'fixed',
            top: 0,
            left: 0,

            marginTop: '5vh',
            marginLeft: '5vh'
          }}
        >
          <ReactPlayer
            style={{
              marginTop: 0,
              padding: 0,
              zIndex: 1800
            }}
            url={focusedVideo}
            playing
            loop
            width='100%'
            height='100%'
          />
        </div>
      </div>
    )
  );
};

export default withWidth({ withTheme: true })(
  connect(
    mapStateToProps,
    mapDispatchToProps
  )(MovieModal)
);
