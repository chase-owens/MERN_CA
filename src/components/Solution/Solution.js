import React from 'react';
import { connect } from 'react-redux';
import { theme } from '../../styles/theme';

import Matricies from '../Matricies/Matricies';
import Typography from '@material-ui/core/Typography';
import withWidth from '@material-ui/core/withWidth';
import { unstable_useMediaQuery as useMediaQuery } from '@material-ui/core/useMediaQuery';

import Translation from '../Translation/Translation';
import { isMobile } from 'react-device-detect';

const mapStateToProps = state => ({
  audience: state.audienceState.audience
});

const Solution = ({ audience }) => {
  const color = useMediaQuery(theme.breakpoints.down(740))
    ? 'textSecondary'
    : 'default';
  return (
    <div
      style={{
        background: useMediaQuery(theme.breakpoints.down(740))
          ? theme.palette.ternary.main
          : '#fff'
      }}
    >
      <div style={{ paddingTop: 20 }}>
        <div style={{ padding: '0 40px' }}>
          <Typography
            style={{ paddingBottom: 10 }}
            color={color}
            paragraph
            variant='caption'
          >
            <span style={{ fontWeight: 'bold' }}>CA</span> / Solution
          </Typography>
          <Typography
            style={{ fontSize: '2em' }}
            color={color}
            paragraph
            variant='h2'
          >
            Scientific approach to establishing calm behaviors
          </Typography>
          <hr
            style={{
              borderColor: useMediaQuery(theme.breakpoints.down(740))
                ? theme.palette.secondary.dark
                : theme.palette.ternary.main,
              marginBottom: 10
            }}
          />
        </div>
        <div style={{ padding: '0 40px' }}>
          <Translation />
        </div>

        {audience === 'gen' ? (
          <div style={{ padding: '0 40px' }}>
            <Typography
              style={{ padding: '20px 0 40px 0', fontSize: '1.2em' }}
              color={color}
              variant='body1'
            >
              We applied state-of-the-art learning science to the development of
              Constructional Affection. Since approaching for affection is a
              member of both excited and calm behaviors, Constructional
              Affection builds upon that foundation to shape and maintain calm
              behaviors. The result is a spoiled companion that now finds being
              calm more reinforcing than getting excited.
            </Typography>
          </div>
        ) : audience === 'ba' ? (
          <div>
            <div style={{ padding: '0 40px' }}>
              <Typography
                style={{ padding: '20px 0 40px 0', fontSize: '1.2em' }}
                color={color}
                variant='body1'
              >
                Constructional Affection is the artifact of following the
                Constructional Approach (Goldiamond, 1974) to create calm
                interactions with dogs. The first step was to define calm
                behaviors and specify under what conditions they would occur.
                Then current relevant repertoire (approaching, sitting, lying
                down) and other assets (affection as a reinforcer) were
                identified. The steps of the program were then defined. The
                first step uses response shaping to expand on the entry
                repertoire and establish calm behaviors. The second step uses
                stimulus fading to maintain calm behaviors under the target
                conditions. Then, if the target objective specifies consequences
                other than petting and scratching, the final step brings the
                target occasion-behavior relation in contact with the target
                consequence.
              </Typography>
            </div>
            {/* <Matricies /> */}
          </div>
        ) : (
          <div style={{ padding: '0 40px' }}>
            <Typography
              style={{
                padding: '20px 0 40px 0',
                fontSize: '1.2em',
                overflow: 'hidden'
              }}
              color={color}
              variant='body1'
            >
              Woof woof. Grrrrr ruff ruff. Bark wiggle yawn bark, bark, bark.
              Woof woof. Grrrrr ruff ruff. Bark, bark-bark-bark, bark-bark-bark,
              bark-bark-bark. Grrrrr ruff ruff. Grrrrr ruff-ruff-ruff-sruff.
            </Typography>
          </div>
        )}

        <br />
      </div>
    </div>
  );
};

export default connect(mapStateToProps)(
  withWidth({ withTheme: true })(Solution)
);
