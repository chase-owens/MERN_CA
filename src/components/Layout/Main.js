import React, { unstable_Profiler as Profiler } from 'react';
import { Helmet } from 'react-helmet';
import NavIntro from './NavIntro';
import About from '../About/About';
import Solution from '../Solution/Solution';
import HowTo from '../HowTo/HowTo';
const Videos = React.lazy(() => import('../Videos/Videos'));
const MovieModal2 = React.lazy(() => import('../MovieModal/MovieModal'));
import Footer from '../Footer/Footer';
import { isMobile } from 'react-device-detect';

const record = (
  id,
  phase,
  actualDuration,
  baseDuration,
  startTime,
  commitTime,
  interactions
) => {
  console.log(
    id,
    phase,
    actualDuration,
    baseDuration,
    startTime,
    commitTime,
    interactions
  );
};

const Main = () => {
  return (
    <div style={{ margin: 0, padding: 0 }}>
      <Helmet>
        <meta charset='UTF-8' />
        <meta
          name='viewport'
          content='width=device-width, initial-scale=1'
        ></meta>
        <title>Constructional Affection</title>
        <meta name='description' content='Constructional Affection' />
      </Helmet>
      <Profiler id='Main' onRender={record}>
        <NavIntro />
        <main style={{ margin: 0, padding: 0 }}>
          <About />
          <Solution />
          <HowTo />
          <React.Suspense fallback={<div />}>
            <Videos />
          </React.Suspense>
          {!isMobile && (
            <React.Suspense fallback={<div />}>
              <MovieModal2 />
            </React.Suspense>
          )}
        </main>
        <Footer />
      </Profiler>
    </div>
  );
};

export default Main;
