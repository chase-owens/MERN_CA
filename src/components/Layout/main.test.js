import React from 'react';
import { shallow, mount } from 'enzyme';
import Main from './Main';
import { store } from '../../app/store';
import { Provider } from 'react-redux';
import { MuiThemeProvider } from '@material-ui/core/styles';
import { theme } from 'styles/theme';

import NavIntro from './NavIntro';
import About from '../About/About';
import Solution from '../Solution/Solution';
import HowTo from '../HowTo/HowTo';
import Videos from '../Videos/Videos';
import MovieModal from '../MovieModal/MovieModal';
import Footer from '../Footer/Footer';
import '../../mocks/matchMediaMock';
// import jsdom from 'jsdom';
// const { JSDOM } = jsdom;
// const doc = new JSDOM(`<!doctype html><html><body></body></html>`);
// global.document = doc;
// global.window = doc.defaultView;

describe('Main', () => {
  it('it did not change visually', () => {
    const main = shallow(<Main />);
    expect(main).toMatchSnapshot();
  });

  // it('renders expected children', () => {
  //   const main = mount(
  //     <Provider store={store}>
  //       <MuiThemeProvider theme={theme}>
  //         <Main />
  //       </MuiThemeProvider>
  //     </Provider>
  //   );
  //   expect(main.find(NavIntro).length).toEqual(1);
  //   expect(main.find(About).length).toEqual(1);
  //   expect(main.find(Solution).length).toEqual(1);
  //   expect(main.find(HowTo).length).toEqual(1);
  //   expect(main.find(Videos).length).toEqual(1);
  //   expect(main.find(MovieModal).length).toEqual(1);
  //   expect(main.find(Footer).length).toEqual(1);
  // });
});
