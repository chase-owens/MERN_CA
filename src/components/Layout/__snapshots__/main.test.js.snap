// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main it did not change visually 1`] = `
<div
  style={
    Object {
      "margin": 0,
      "marginRight": 0,
      "padding": 0,
    }
  }
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <meta
      charset="UTF-8"
    />
    <title>
      Constructional Affection
    </title>
    <meta
      content="Constructional Affection"
      name="description"
    />
  </HelmetWrapper>
  <WithTheme(WithWidth(Connect(NavIntro))) />
  <main
    style={
      Object {
        "margin": 0,
        "padding": 0,
      }
    }
  >
    <WithTheme(WithWidth(About)) />
    <Connect(WithTheme(WithWidth(Solution))) />
    <WithTheme(WithWidth(HowTo)) />
    <WithStyles(WithTheme(WithWidth(Videos))) />
    <WithTheme(WithWidth(Connect(MovieModal))) />
  </main>
  <WithStyles(Footer) />
</div>
`;
